wagtail/admin/wagtail_hooks.py

```diff
@@ -460,6 +447,29 @@ def page_header_buttons(page, page_perms, is_parent=False, next_url=None):
             },
             priority=70,
         )
+    if (
+        page_perms.user.has_perm("simple_translation.submit_translation")
+        and not page.is_root()
+    ):
```

Comment:
maybe run the tests with this commented out temporally (locally) - most likely this is the cause of the extra query and that should be ok.

But be sure that you are confident the extra query is not an accident.


```
FAIL: test_page_edit_num_queries (wagtail.admin.tests.pages.test_edit_page.TestPageEdit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/project/wagtail/admin/tests/pages/test_edit_page.py", line 1282, in test_page_edit_num_queries
    self.client.get(
  File "/home/circleci/project/.venv/lib/python3.8/site-packages/django/test/testcases.py", line 100, in __exit__
    self.test_case.assertEqual(
AssertionError: 39 != 38 : 39 queries executed, 38 expected
```

Reply:
This should ideally check if the `simple_translation` app is enabled, then perform the permission check.

There's an argument about better caching permission checks on the request, but that's a different story

Reply:
note: the simple translation permissions will only exist IF the app is installed.